_posts/2018-09-28-3D-Printing-Intro.md

```diff
@@ -0,0 +1,400 @@
+    </a>
+</div>
+<br/>
+
+#### SLA & DLP: Stereolithography & Digital Light Processing
+
+<img alt="stack" src="{{ '/img/2018-09-28-3D-Printing/sla-dlp.jpeg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 800px;">
+
+These types of printers work by using either a laser or a projected image to cure a UV-reactive resin.
+
+With `Stereolithography` a powerful UV laser traces the object layer by layer, like the FDM printer does, however it does not deposit material itself, rather it cures the resin in the tank at the point where the laser is.
+Because SLA uses a laser that is moved by mirrors, it can have a very high resolution, the disadvantage is that it is as slow as an FDM printer since each layer needs to be traced.
+
+With `Digital Light Processing` an UV projector is used, it projects the an entire layer at once, the results in a lower printable resolution for the object, but yields a significant speed increase since an entire layer is printed at once, there is no need to trace the entire layer.
+`Masked SLA` is a cheaper version of the DLP method where a LCD based photomask is used in front of an UV-LED array instead of an UV based projector.
+
+<img alt="stack" src="{{ '/img/2018-09-28-3D-Printing/sla-dlp-2.jpeg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 800px;">
+
+<img alt="stack" src="{{ '/img/2018-09-28-3D-Printing/sla-vs-dlp.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 800px;">
+
+The images above show the distinct difference between these two (three) methods.
+The SLA method has much better quality compared to the DLP or MSLA methods.
+
+<div style="text-align: center; margin: 0px auto;">
+    <a href="{{ '/img/2018-09-28-3D-Printing/sla-print1.jpg' | prepend: site.baseurl }}" data-lightbox="fdm" data-title="Multiple eiffel towers printed at once with DLP/MSLA">
+        <img alt="stack" src="{{ '/img/2018-09-28-3D-Printing/sla-print1.jpeg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; width: 35%; display: inline-block;">
+    </a>
+    <a href="{{ '/img/2018-09-28-3D-Printing/sla-print2.jpg' | prepend: site.baseurl }}" data-lightbox="fdm" data-title="Vase print with SLA">
+        <img alt="stack" src="{{ '/img/2018-09-28-3D-Printing/sla-print2.gif' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; width: 42%; display: inline-block;">
+    </a>
+</div>
+<br/>
+
+#### DMLS & SLM: Direct Metal Laser Sintering & Selective Laser Melting
+
+<img alt="stack" src="{{ '/img/2018-09-28-3D-Printing/dmls-slm.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 500px;">
+
+`Direct Metal Laser Sintering` and `Selective Laser Melting` are two more industrial types of 3D printing. They are used, as the name suggests, to 3D print object in metal.
```

Comment:
are two more industrial types => are two of the more industrial methods

(not sure about replacing types by methods, but methods sounds more logical)